doc/nrf-bm/release_notes/release_notes_1.0.0.rst

```diff
@@ -0,0 +1,486 @@
+.. _nrf_bm_release_notes_100:
+
+Release Notes for |BMlong| v1.0.0
+#################################
+
+|BMshort| v1.0.0 is the first supported release of this product, demonstrating a set of samples for the `nRF54L15 DK`_ with build targets supporting the following SoCs:
+
+* `nRF54L15`_
+* `nRF54L10`_ (emulated on the nRF54L15 DK)
+* `nRF54L05`_ (emulated on the nRF54L15 DK)
+
+The supported DK version is `nRF54L15 DK v1.0.0`_.
+
+This release includes two versions of the SoftDevice - a Bluetooth® LE stack to be used on the nRF54L15 DK:
+
+* S115 - Support for peripheral role only
+* S145 - Support for both peripheral and central role, as well as more connections
+
+Highlights
+**********
+
+The overall software maturity status of |BMlong| v1.0.0 is supported unless stated differently for individual features or components.
+This makes it suitable for product development using nRF54L15, nRF54L10, and nRF54L05 Wireless SoCs, including the nRF54L15 DK targets.
+
+Added the following features as supported:
+
+* The S115 SoftDevice, which was part of |BMshort| v0.9.0 as experimental, is now supported.
+  The qualification for this SoftDevice variant will be made available in the compatibility matrix for nRF54L15, nRF54L10, and nRF54L05, by the end of 2025.
+
+* The S145 SoftDevice, a new variant that enables central and multirole operation, has been added as supported.
+  The qualification for this SoftDevice variant will be made available in the compatibility matrix for nRF54L15, nRF54L10, and nRF54L05, by the end of 2025.
+
+* MCUBoot (IRoT)/DFU:
+
+  * CRACKEN/KMU is now supported.
+  * Possible to be configured as IRoT for your applications.
+  * Available in two variants: debug and size-optimized.
+
+* New samples and libraries.
+  Refer to the changelog for detailed information.
+
+Added the following features as experimental:
+
+* NFC Type 4 Tag and Type 2 Tag libraries, including samples.
+
+* Bluetooth LE Central functionality as part of samples and libraries utilizing the S145 SoftDevice.
+
+  .. note::
+     While the S145 SoftDevice has supported maturity, its overall integration in the SDK is experimental.
+
+Release tag
+***********
+
+The release tag for the |BMshort| manifest repository is **v1.0.0**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
```

Comment:
This was very Command line oriented, could it be adjusted to cover VSC as well (Install SDK, no need for West update)?